base-images/remote-cache/build-and-push-remote-cache.sh

```diff
@@ -11,6 +11,8 @@ DOCKER_HUB_ORG="${DOCKER_HUB_ORG:-tensorchord}"
 cd ${ROOT_DIR}
 
 envd build --export-cache type=registry,ref=docker.io/${DOCKER_HUB_ORG}/python-cache:envd-v${ENVD_VERSION} --force
-envd build -f build.envd:build_gpu --export-cache type=registry,ref=docker.io/${DOCKER_HUB_ORG}/python-cache:envd-v${ENVD_VERSION} --force
+envd build -f build.envd:build_gpu_11_2 --export-cache type=registry,ref=docker.io/${DOCKER_HUB_ORG}/python-cache:envd-v${ENVD_VERSION} --force
+envd build -f build.envd:build_gpu_11_3 --export-cache type=registry,ref=docker.io/${DOCKER_HUB_ORG}/python-cache:envd-v${ENVD_VERSION} --force
+envd build -f build.envd:build_gpu_11_6 --export-cache type=registry,ref=docker.io/${DOCKER_HUB_ORG}/python-cache:envd-v${ENVD_VERSION} --force
```

Comment:
Will those caches overwrite each other? Have you tested it locally?

Reply:
It will be overridden. We should:

- Generate different image caches with different tags. For example, docker.io/tensorchord/python-cache:envd-vxx-cuda-11.2
- Update the code about the cache importer https://github.com/tensorchord/envd/blob/main/pkg/lang/ir/compile.go#L189 to use the expected cache.

Reply:
Fixed. PTAL.